cycode/cli/consts.py

```diff
@@ -189,3 +189,5 @@
 # Example: A -> B -> C
 # Result: A -> ... -> C
 SCA_SHORTCUT_DEPENDENCY_PATHS = 2
+
+SCA_DISABLE_RESTORE_DEPENDENCIES_FLAG = 'disable-restore-dependencies'
```

Comment:
looks so complecated to type for users. should we at least add shortcut? 

we have -o for --option
-t for --scan-type 

and etc

Reply:
I'll consult with my product